oak_runtime/src/lib.rs

```diff
@@ -340,6 +340,13 @@ pub enum LabelReadStatus {
     NeedsCapacity(usize),
 }
 
+/// Indicator whether an operation is executed using the Node's privilege or without it.
+#[derive(Clone, Copy, Debug)]
+enum OperationPrivilege {
+    None,
+    UsePrivilege,
+}
```

Comment:
Similarly to my previous comment, maybe just `Downgrading { No, Yes }` or similar?

Reply:
Done.